src/olympia/devhub/templates/devhub/email/revoked-key-email.ltxt

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}{% blocktrans %}The API Key was revoked from your account:
+
+{{ key }}
+
+If you believe this key was revoked in error, you can create a new one
+at the following location:
+
+{{ url('devhub.api_key')|absolutify }}
+{% endblocktrans %}
```

Comment:
If I'm not mistaken this template is being interpreted as a jinja-template via jingo and jinja doesn't have `blocktrans`. You could theoretically do something like:

```html+jinja
{% trans %}The API Key was revoked from your account:

{{ key }}

If you believe this key was revoked in error, you can create a new one
at the following location:

{{ url('devhub.api_key')|absolutify }}
{% endtrans %}
```